src/Doctrine/Mapping/ClassMetadataFactory.php

```diff
@@ -29,7 +29,7 @@ public function __construct(string $tmpDir)
 	protected function initialize(): void
 	{
 		$drivers = [];
-		if (class_exists(AnnotationReader::class)) {
+		if (class_exists(AnnotationDriver::class)) {
```

Comment:
Should probably ask for both?

Reply:
`doctrine/annotations` is peer dependency for `doctrine/orm` since orm version `2.10.0`

but if you want to use AnnotationDriver, it's "necessary"

on the other hand, why not add a double check 👍 

give me sec